app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError, NotWearingMaskError)
```

Comment:
In case you place imports on the same line, there is no need to place them inside of `()`.